File: This is for Python/BMI.py
#Hello, we are going to make a simple BMI calculator

#We'll add name for aesthetic purposes
def name(name):
    return(name)

#assigning value to variable 'user_name' by using input
user_name = (input("What is your name? "))

#This assigns the define function with the 'user_name' parameter to variable Username
Username = name(user_name)

print("Hello,", Username,".......Welcome to the BMI calculator created using python")

#first we need to add a define function 
#(height) and (weight) are called parameters in the function
def bodymassindex(weight,height):
    return round((weight / height**2),2)

#getting input for user using variable
#since we're putting inputs in a string format, we'll convert them into floats to allow us to perform mathematical operations using the variables
w = float(input("Enter your weight(in kilograms):  "))
h = float(input("Enter your height (in meters): "))


#we'll use a function call to bodymassindex with (h) and (w) as arguments 
#we'll assign the function call to the variable (bmi)
bmi = bodymassindex(w,h)
print('Your BMI is:', bmi)

#we'll use the conditions 
if bmi <= 18.5:
    print("You're underweight,", Username)
elif 18.6 < bmi < 24.99:
    print("You're BMI is Normal,", Username)
elif 25 < bmi < 29.29:
    print("You're overweight,", Username)
else:
    print("You're obese,", Username)

File: This_is_for_C++/BMI.cpp
/*
Activity: BMI calculator
Description: Using cmath library for power function; usage of if-else statements
*/

#include <iostream> //include header library for cout
#include <cmath> //include library for power function

using namespace std;

int main(){

    char name[20];//variable 1
    int input;//variable 2
    float height;//variable 3
    float weight;//variable 4
    float result;//variable 5


    cout << "What's your name?   ";
    cin >> name;//assigning value to variable name
    cout << "Hello,"<<name<<". This is a simple BMI calculator.\nWould you like to try the sample?\n";
    cout << "1 - Yes\n";
    cout << "2 - No";
    cout << "\nYour answer:   ";
    cin >> input;

    if (input == 1){
        cout<<"Enter your height(in meters):   ";
        cin>>height; //inputting value in variable height

        cout<<"Enter your weight(in kilograms):   ";
        cin>>weight;// inputting value in variable weight

        int power = pow(height, 2); //power function allows us to square 
            //power is a variable
             //pow(x,y) is the funtion that takes parameters
             // x is the integer
             // y is the power 

        result = weight/power; // assigning value to result 

        cout<<"Your BMI is: "<<result<<".";

            //using conditions for BMI results
            if (result <= 18.5){
                cout << "\nYou're underweight";
            }
            else if (18.6 < result < 24.99){
                cout << "\nYour BMI is normal";
             }    
            else if (25 < result < 29.29){
                cout << "\nYou're overweight";
            } 
            else{
                cout << "\nYou're obese";
            }
    }
    else if (input == 2){
        cout<<"Okay,"<<name<<". See you next time.";
    }
    else{
        cout << "INVALID";
    }
return 0;
}